salt/pillar/ec2_pillar.py

```diff
@@ -14,6 +14,9 @@
 of the instances to specific hosts.  There is no glob matching at
 this time.
 
+  Note: If using `use_grain: True` minion must have `metadata_server_grains: True` set
+  in minion config file.
```

Comment:
Please put a link to the meta data grain. Maybe with
 ```
:mod:`ec2 metadata <salt.grains.metadata>`
```
Maybe some from the salt team can provide a hint.